src/components/combo_box/combo_box_input/combo_box_input.tsx

```diff
@@ -150,6 +152,8 @@ export class EuiComboBoxInput<T> extends Component<
       append,
       isLoading,
       autoFocus,
+      ariaLabel,
+      ariaLabelledby,
```

Comment:
Maybe I'm missing something, but why not just use the native `aria-label` prop instead of our own camel-cased `ariaLabel`? I would rather see us use native props whenever possible.

We can still destructure them with camelCasing like so:

```suggestion
      'aria-label': ariaLabel,
      'aria-labelledby': ariaLabelledby,
```

Reply:
I think I've got a language and/or intent problem with these prop names. They originate in the `combo_box.tsx` file and depend on users entering a descriptive string for `ariaLabel` or referencing an ID string for `ariaLabelledby`. The parent component is just a pass through; the native attributes aren't rendered until lines 287 and 288.

I prefer to keep prop names custom until I'm ready to render the native attribute to the DOM, which is why I went this direction. Would it be better to rename these props to something more descriptive? Maybe `accessibleLabel` and `accessibleLabelId` ?

Reply:
No, the issue is one of consistency across our codebase and for our consumers. Thus far we have primarily made use of the native `aria-label` and `aria-labelledby` APIs/props (just do a grep across the codebase, you will notice them similarly destructured as my example above) and consumers expect it at this point. To suddenly deviate from that for a single component is going to be super confusing and inconsistent, just IMO.

I'm not quite understanding your objection re: intent - a dev/consumer is going to pass us an aria-label for an arbitrary component and expect us to place it in the appropriate location that makes it work for accessibility and screen readers - they don't actually care about the semantics or details (which is one of the points of a component library IMO, to abstract away minutiae).

It's really not the worst thing in the world to just do something like this:

```jsx
const ParentComponent = (props) => {
  const {
    'aria-label': ariaLabel,
    ...
  } = props;

  return <ChildComponent aria-label={ariaLabel} />
};

const ChildComponent = (props) => {
  const {
    'aria-label': ariaLabel,
    ...
  } = props;

  return <input aria-label={ariaLabel} />;
};
```

It feels a little redundant, but it really isn't the worst thing in the world, and the problem is one of deviating away from the majority of the rest of our components.